torch/jit/_script.py

```diff
@@ -1270,7 +1270,7 @@ def forward(self, a) -> MyModule:
         return create_script_dict(obj)
     if isinstance(obj, list):
         return create_script_list(obj)
-
+    isoverloadpacket = isinstance(obj, torch._ops.OpOverloadPacket)
```

Comment:
Why is this logic needed? Is there plan to directly script an (overloaded) op?

Reply:
I looked up all instances in TorchScript where it was being checked if an object is a function or a method and added handling for OverloadPacket (to do the same thing you'd do if you received an op as before), just as a precaution. I can remove it if you think it's not needed

Reply:
Yeah, I don't think this would be needed, Let's try remove it and also add some test cases where per-overload op is used in TorchScript to see if everything still works.